src/Microsoft.Data.Sqlite/project.json

```diff
@@ -9,10 +16,7 @@
     ],
     "xmlDoc": true,
     "embed": {
-      "include": [
-        "Properties/Microsoft.Data.Sqlite.rd.xml",
-        "Strings.resx"
```

Comment:
Discussed offline and we don't know why this was ever there in the first place. :blush: